spec/tasks/ingest_spec.rb

```diff
@@ -28,10 +28,20 @@ def loaded_files_excluding_current_rake_file
       "hearing over eminent domain for construction of Harbor Freeway, Calif., 1947"
     end
     let(:description) { ["At the Hall of Records, 220 N. Broadway.", "Description 2"] }
+    let(:subj) do
+      ["Express highways--California--Los Angeles County--Design and construction",
+       "Eminent domain--California--Los Angeles",
+       "Demonstrations--California--Los Angeles County",
+       "Transportation",
+       "Government",
+       "Activism",
+       "Interstate 10"]
+    end
 
     it 'has created a work with the correct metadata' do
       expect(created_work.title).to contain_exactly(title)
       expect(created_work.description).to contain_exactly(*description)
+      expect(created_work.subject).to contain_exactly(*subj)
```

Comment:
We might want to change this eventually to `expect(created_work).to have_attributes(..., subject: contain_exactly(*subj), ...)`. That would report unexpected values all at once, instead of failing at the first bad expectation.

For now, I'm :+1: on this.